frontend/src/app/features/auth/sign-in/sign-in.component.html

```diff
@@ -0,0 +1,31 @@
+<form class="sign-in-wrapper" [formGroup]="signInGroup">
+    <h1 i18n>Log into your account</h1>
+    <mat-form-field class="sign-in-field">
+        <mat-label i18n>Email</mat-label>
+        <input formControlName="email" matInput type="text" (keydown)="updateLoginErrors()"/>
```

Comment:
updateLoginErrors - i guess validate is more simple naming

Reply:
Fixed